<?xml version="1.0"?>
<launch>
  <group ns="guzheng">
    <node name="string_fitter" pkg="tams_pr2_guzheng" type="fit_strings.py" />
  </group>
</launch>
